cli/pkg/backend/aws/backend.go

```diff
@@ -296,3 +301,35 @@ func (b *Backend) GetEvents(ctx context.Context, stackName string, services []st
 func (b *Backend) Describe(ctx context.Context, stackName string, serviceName string) (compute.StackServiceDescription, error) {
 	return b.ComputeBackend.Describe(ctx, stackName, serviceName)
 }
+
+func (b *Backend) DisplayCloudWatchInsightsLink(ctx context.Context, logReference util.LogReference) error {
+	queryId := uuid.NewUUID()
+	cloudwatchLink, err := util.LogInsights2ConsoleLink(logReference,
+		string(queryId))
+	if err != nil {
+		logrus.Errorf("To our dismay, we were unable to generate a link to query and visualize these logs")
+	} else {
+		if diagnostics.IsInteractiveContext(ctx) {
+			proceed := false
+			prompt := &survey.Confirm{Message: fmt.Sprintf("Would you like to query these logs in your browser? Please log into your AWS account (%s), then select Yes.", logReference.AWSAccountID)}
+			err = survey.AskOne(prompt, &proceed)
+			if err != nil || !proceed {
+				return nil
+			}
+			logrus.Info("Opening Browser window to query cloudwatch insights.")
+			err = browser.OpenURL(cloudwatchLink)
+			if err != nil {
+				return errors.Wrap(err, "To our dismay, we were unable open up a browser window to query cloudwatch insights.")
+			}
+			logrus.Info("Select the desired time frame, and click 'Run Query' to query the logs.")
+			return nil
+		}
+		logrus.Info("****************************************************************************************")
```

Comment:
would you mind showing what this looks like in the terminal and put it in the PR description

Reply:
Done.